donate/thunderbird/templates/pages/core/contributor_support_page.html

```diff
@@ -1,5 +1,20 @@
 {% extends "pages/core/contributor_support_page_master.html" %}
 
+
+{% block donate_support_text %}
+    <p class="rich-text">
+        {% blocktrans trimmed %}
```

Comment:
Actually, this is broken due to the missing `{% load i18n %}` at the beginning of the template
<img width="1086" alt="Capture d’écran 2021-05-12 à 19 07 39" src="https://user-images.githubusercontent.com/1294206/118016136-f38e2e80-b34c-11eb-9789-1f83230e9442.png">